src/coreclr/tests/src/Common/Directory.Build.targets

```diff
@@ -84,6 +84,28 @@
           Include="$(CoreCLRArtifactsPath)%(RunTimeArtifactsIncludeFolders.Identity)**/*"
           Exclude="@(RunTimeArtifactsExcludeFiles -> '$(CoreCLRArtifactsPath)%(Identity)')"
           TargetDir="%(RunTimeArtifactsIncludeFolders.Identity)" />
+
+      <RunTimeDependencyCopyLocal
+          Condition="'$(TargetArchitecture)' == 'wasm'"
+          Include="$(ArtifactsBinDir)microsoft.netcore.app.runtime.$(TargetOS)-$(TargetArchitecture)\$(Configuration)\runtimes\$(TargetOS)-$(TargetArchitecture)\**"
```

Comment:
You need to use $(TargetOS.ToLower()) in both places, the name contains lowercase "browser" in it.